### PR TITLE
Return all user not found errors

### PR DESCRIPTION
**Pull request by kitagry**
Hi @snowhork. Thank you for the nice tool!

I want all users which don't create account yet. So, I changed to return multiple errors.

Thank you!

### CODE REVIEW

**Comment by snowhork** [comment]
@kitagry Great work. It looks ok. 

Could you give me a sample screenshot for multierr message? Please.

**Comment by kitagry** [comment]
like following. (Sorry. I couldn't screenshot because there is some secret information :pray:)

```bash
$ rdiam add -y -u not-found1@example.com -u not-found2@example.com -g hoge
users:   [not-found1@example.com not-found2@example.com]
groups:  [hoge]
Error: findUserID: user: "not-found1@example.com" not found; findUserID: user: "not-found2@example.com" not found
Usage:
  rdiam add -u user1@email.com,user2@email.com -g group1,group2 [flags]

Flags:
  -g, --groups strings   Specify group(s)
  -h, --help             help for add
  -u, --users strings    Specify user email(s)
  -y, --yes              Automatic yes to prompts

Global Flags:
      --config string   config file (default is $HOME/.rdiam.yaml)
  -v, --verbose         print verbose log
```

**Comment by kitagry** [comment]
Hmm, test is failed. I'll fix it.

**Comment by kitagry** [comment]
I think it is because `go.uber.org/atomic` uses the generics. Do you mind that I upgrade go version to 1.18? If you don't want to do, I'll downgrade `go.uber.org/multierr` version.

```
  level=warning msg="[runner] Can't run linter goanalysis_metalinter: goanalysis_metalinter: buildir: package \"atomic\" (isInitialPkg: false, needAnalyzeSource: true): T"
```

**Comment by snowhork** [comment]
@kitagry No problem. Please upgrade.

File: cmd/root.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```
Comment (kitagry) [inline cmd/root.go:8]: gofmt(with Go1.19) format comments.

**Comment by snowhork** [comment]
@kitagry It looks good. Is it ready to merge?

**Review by snowhork — APPROVED** [review]

**Comment by kitagry** [comment]
Yes!Thank you!

**Pull request merged**